src/main/java/bio/terra/service/filedata/azure/util/BlobContainerClientFactory.java

```diff
@@ -169,9 +174,44 @@ private String createReadOnlySasUrlForBlobUsingClient(String blobName) {
         "%s/%s?%s", blobContainerClient.getBlobContainerUrl(), blobClient.getBlobName(), sasToken);
   }
 
-  private BlobServiceSasSignatureValues createSasSignatureValues() {
+  public enum SASPermission {
```

Comment:
I think case-wise we want this to be:
```suggestion
  public enum SasPermission {
```